lib/parse-model.js

```diff
@@ -11,20 +11,26 @@ function configureFaker(config) {
 function parseModel(model) {
   const modelKeys = Object.keys(model)
   const parsedModel = modelKeys.reduce((accumulator, currentValue) => {
-    const { type, value } = model[currentValue]
-
-    const _value = modelAttributeTypes[type](value)
+    const { type, value, options } = model[currentValue]
+    const _value = modelAttributeTypes[type](value, options)
 
     const _accumulator = Object.assign(accumulator, { [currentValue]: _value })
     return _accumulator
   }, {})
   return parsedModel
 }
 
+function parseArray(model, options) {
+  return [...Array(options.size).keys()].map(() => {
+    return parseModel(model.value)
```

Comment:
This is breaking the next iteration of parseModel, I think the correct param should be just **model**. We only need a parseArray function to iterate *n* times, we don't need to destructure the param.